lib/modules/versioning/ubuntu/common.ts

```diff
@@ -1,28 +1,40 @@
 import { regEx } from '../../../util/regex';
 
+const regex = regEx(/^(?<codename>\w+)-(?<date>\d{8})(?<suffix>\.\d{1,2})?$/);
+
 function isDatedCodeName(input: string): boolean {
-  return regEx(/^(?<codename>\w+)-(?<date>\d{8})$/).test(input);
+  return regex.test(input);
 }
 
 function getDatedContainerImageCodename(version: string): null | string {
-  const groups = regEx(/^(?<codename>\w+)-(?<date>\d{8})$/).exec(version);
+  const groups = regex.exec(version);
   if (!groups?.groups) {
     return null;
   }
   return groups.groups.codename;
 }
 
 function getDatedContainerImageVersion(version: string): null | number {
-  const groups = regEx(/^(?<codename>\w+)-(?<date>\d{8})$/).exec(version);
+  const groups = regex.exec(version);
   if (!groups?.groups) {
     return null;
   }
 
   return parseInt(groups.groups.date, 10);
 }
 
+function getDatedContainerImageSuffix(version: string): null | string {
+  const groups = regex.exec(version);
+  if (!groups?.groups) {
```

Comment:
```suggestion
  if (!groups?.groups?.suffix) {
```

Otherwise this function can return `undefined`